be/src/exec/orc_scanner.cpp

```diff
@@ -0,0 +1,261 @@
+                std::cout << "rows of group "<< _rows_of_group << std::endl;
+                _batch = _row_reader->createRowBatch(_rows_of_group);
+                _row_reader->next(*_batch.get());
+
+                _current_line_of_group = 0;
+                ++_current_group;
+            }
+
+            std::vector<orc::ColumnVectorBatch *> _batch_vec = ((orc::StructVectorBatch *) _batch.get())->fields;
+            for (int column_ipos = 0; column_ipos < _num_of_columns_from_file; ++column_ipos) {
+                auto slot_desc = _src_slot_descs[column_ipos];
+                orc::ColumnVectorBatch *cvb = _batch_vec[_column_name_map_orc_index.find(
+                        slot_desc->col_name())->second];
+
+                if (cvb->hasNulls && !cvb->notNull[_current_line_of_group]) {
+                    if (!slot_desc->is_nullable()) {
+                        std::stringstream str_error;
+                        str_error << "The field name(" << slot_desc->col_name()
+                                  << ") is not allowed null, but ORC field is NULL.";
+                        LOG(WARNING) << str_error.str();
+                        return Status::RuntimeError(str_error.str());
+                    }
+                    _src_tuple->set_null(slot_desc->null_indicator_offset());
+                } else {
+                    int32_t wbytes = 0;
+                    uint8_t tmp_buf[128] = {0};
+                    _src_tuple->set_not_null(slot_desc->null_indicator_offset());
+                    void *slot = _src_tuple->get_slot(slot_desc->tuple_offset());
+                    StringValue *str_slot = reinterpret_cast<StringValue *>(slot);
+
+                    switch (_reader->getType().getSubtype(column_ipos)->getKind()) {
+                        case orc::BOOLEAN: {
+                            int64_t value = ((orc::LongVectorBatch *) cvb)->data[_current_line_of_group];
+                            if (value == 0) {
+                                str_slot->ptr = reinterpret_cast<char *>(tuple_pool->allocate(5));
+                                memcpy(str_slot->ptr, "false", 5);
+                                str_slot->len = 5;
+                            } else {
+                                str_slot->ptr = reinterpret_cast<char *>(tuple_pool->allocate(4));
+                                memcpy(str_slot->ptr, "true", 4);
+                                str_slot->len = 4;
+                            }
+                            break;
+                        }
+                        case orc::INT:
+                        case orc::SHORT:
+                        case orc::LONG:
+                        case orc::DATE: {
+                            int64_t value = ((orc::LongVectorBatch *) cvb)->data[_current_line_of_group];
+                            std::cout << "value : " << value << std::endl;
+                            wbytes = sprintf((char *) tmp_buf, "%ld", value);
```

Comment:
does it work for date type?

Reply:
orc::DATE is only day such as "20191111" , date time type is orc::TIMESTAMP. I will support in next PR